src/common/components/hosts/HostRowDetail.tsx

```diff
@@ -29,6 +29,12 @@ type HostDetailProps = {
   onDiskRole?: onDiskRoleType;
   AdditionalNTPSourcesDialogToggleComponent?: ValidationInfoActionProps['AdditionalNTPSourcesDialogToggleComponent'];
   hideNTPStatus?: boolean;
+  showStorage?: boolean;
+  updateDiskSkipFormatting?: (
```

Comment:
We could extract this definition as it's used in a few places.
We're doing it for `onDiskRoleType` for example.